### ISSUE TITLE
Bug when formatting selection at top of file

### ISSUE THREAD

**Issue by rupert** [rupert/pyls-black#1]
The following gets formatted correctly:

```python
# hello
x=1+2
```

If the comment is removed the code is not formatted:

```python
x=1+2
```

Tested using Sublime Text.

**Comment by rupert** [comment]
* Sublime Text `3.1.1 Build 3176`
* LSP `0.6.0`

```
appdirs==1.4.3
attrs==18.1.0
black==18.6b2
click==6.7
future==0.16.0
isort==4.3.4
jedi==0.12.0
mypy==0.610
parso==0.2.1
pluggy==0.6.0
pyls-black==0.1.0
pyls-isort==0.1.1
pyls-mypy==0.1.3
python-language-server==0.19.0
toml==0.9.4
typed-ast==1.1.0
```

**Comment by rupert** [comment]
Fixed in `0.2.0`.
